.prettierignore

```diff
@@ -39,6 +36,9 @@ tsconfig.tsbuildinfo
 # Metadata Files
 CODEOWNERS
 
+# Public Folders
+apps/site/public
+
```

Comment:
Why? We can still make these look pretty?

Reply:
agreed

Reply:
Hmmm, I don't think formatting SVG icons that aren't React components really matter.

Reply:
No strong opinions, so I'll resolve this